docker/api/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
It looks like this file is present twice.  So there is a question of where this file should live.  There should probably only be one copy of it and then we can move it to where it is needed (if different) during the build process.

Reply:
Resolved by deleting the filed in the API container.